app_guides/php/getting_started.adoc

```diff
@@ -0,0 +1,297 @@
+[[appguides-php-getting-started]]
+= Getting Started with PHP
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+=== Overview
+
+This tutorial walks through the creation of a PHP application titled `myapp`.
+Before going through the rest of the tutorial, be sure to have created an
+ifdef::openshift-online[]
+https://console.preview.openshift.com[{product-title} account]
+endif::[]
+ifdef::openshift-origin,openshift-enterprise,openshift-dedicated[]
+{product-title} account
+endif::[]
+and installed the `oc` command line tools.
```

Comment:
A prerequisites step might be more visible than having that info in the overview.